Automattic/jetpack
# Social | Social post UI: Update connection toggle and media notice

## Proposed changes:

* Make the social post Instagram notice same as the one in sidebar
* Social post modal: Hide toggle if connection or media is not valid
* Move social post connection toggle to the top and update the label

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Enable the editor preview feature - `define( 'JETPACK_SOCIAL_HAS_EDITOR_PREVIEW', true );`
* Goto post editor
* Open Jetpack or Social sidebar
* Under "Share this post" section, click on "Create custom posts"
* Confirm that the connection toggle is at the top before the preview
* Confirm that no media notice for  Instagram is same as the one in sidebar
* Confirm that the toggle is hidden when there is a validation notice for Instagram.
* Confirm that the toggle label says, "Connection enabled"

<img width="931" alt="Screenshot 2024-08-09 at 3 21 10 PM" src="https://github.com/user-attachments/assets/c056b3ee-9001-41d6-a00c-725c964a3c20">

<img width="929" alt="Screenshot 2024-08-09 at 3 22 19 PM" src="https://github.com/user-attachments/assets/dc54e3db-231c-4d88-93b9-719af48e488e">
